pkg/plugins/events/sns/sns.go

```diff
@@ -39,16 +41,16 @@ type SnsEventService struct {
 	provider  core.AwsProvider
 }
 
-func (s *SnsEventService) getTopics() (map[string]string, error) {
+func (s *SnsEventService) getTopics(ctx context.Context) (map[string]string, error) {
 	return s.provider.GetResources(core.AwsResource_Topic)
 }
 
-func (s *SnsEventService) getStateMachines() (map[string]string, error) {
+func (s *SnsEventService) getStateMachines(ctx context.Context) (map[string]string, error) {
```

Comment:
ctx unused?

Reply:
i'll pass this down into the resourcesGet